code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/AepCardUIState.kt

```diff
@@ -12,12 +12,16 @@
 package com.adobe.marketing.mobile.aepcomposeui.aepui.state
 
 /**
- * Interface representing the state of an AEP card.
+ * Class representing the state of an AEP card.
  *
- * This interface includes common properties like `dismissed` and `selected`,
- * which can be shared across different card states.
+ * This class includes the common properties `dismissed`, `selected`, and `read` which are common across different card states.
+ *
+ * @property dismissed Indicates whether the card has been dismissed.
+ * @property selected Indicates whether the card is selected.
+ * @property read Indicates whether the card has been read.
  */
-interface AepCardUIState {
-    val dismissed: Boolean
-    val selected: Boolean
-}
+open class AepCardUIState(
+    open val dismissed: Boolean = false,
+    open val selected: Boolean = false,
+    open val read: Boolean = false
```

Comment:
Don't think we need these for the first iteration. Read state will be supported when we add inbox capability